src/web/src/pages/SubjectExplorer.vue

```diff
@@ -143,33 +119,34 @@ export default {
       ],
     };
   },
-  methods: {},
-  computed: {
-    //courseColumns[0] corresponds to left column, [1] to right column
-    courseColumns() {
-      let leftColumn = [];
-      let rightColumn = [];
-      const courses = this.$store.state.courseList;
-      //Obtain All Courses Such That Department Matches The Subject Name.
-      const allTempData = courses.filter((c) => c.department === this.subject);
-      for (let k = 0; k < allTempData.length; k++) {
-        if (k % 2 == 0) leftColumn.push(allTempData[k]);
-        else rightColumn.push(allTempData[k]);
-      }
-      return [leftColumn, rightColumn];
-    },
+
```

Comment:
We probably don't want to change this. These changes were made part of #337. It would probably be best to leave it as is as much as possible